js/client-topbar.js

```diff
@@ -44,7 +44,12 @@
 			app.addPopup(LoginPopup);
 		},
 		openSounds: function () {
-			app.addPopup(SoundsPopup);
+			if (!app.addPopup(SoundsPopup)) {
```

Comment:
This seems to happen whenever the sound button is closed, rather than only on double-click?

Reply:
I can measure the time interval between the two clicks if you care to tell me what interval you think is appropriate...

Reply:
You can also just sidestep manually time-intervaling by listening to the `doubleclick` event.

Reply:
I found out where I originally went wrong with that approach - it's called `dblclick`.